### PR TITLE
user guide of kmeshctl

### PR DESCRIPTION
**Pull request by YaoZengzeng**

### CODE REVIEW

**Comment by YaoZengzeng** [comment]
@hzxuzhonghu PTAL

**Comment by YaoZengzeng** [comment]
ping @hzxuzhonghu

File: content/en/docs/userguide/use_kmeshctl.md [hunk 1]
```diff
@@ -0,0 +1,239 @@
+---
+draft: false
+linktitle: Use kmeshctl
```
Comment (hzxuzhonghu) [inline content/en/docs/userguide/use_kmeshctl.md:3]:
```suggestion
linktitle: Kmeshctl Usage
```

File: content/en/docs/userguide/use_kmeshctl.md [hunk 2]
```diff
@@ -0,0 +1,239 @@
...
+
+```bash
+kmeshctl accesslog [flags]
+```
+
+Examples:
+
+```bash
+# Enable Kmesh's accesslog:
+kmeshctl accesslog <kmesh-daemon-pod> enable
```
Comment (hzxuzhonghu) [inline content/en/docs/userguide/use_kmeshctl.md:58]: <kmesh-daemon-pod> is optional, if ommitted, all accesslog enabled

File: content/en/docs/userguide/use_kmeshctl.md [hunk 3]
```diff
@@ -0,0 +1,239 @@
...
+kmeshctl waypoint apply
+
+# Generate a waypoint as yaml
+kmeshctl waypoint generate --namespace default
+
+# List all waypoints in a specific namespace
+kmeshctl waypoint list --namespace default
+```
+
+#### kmeshctl waypoint apply
```
Comment (hzxuzhonghu) [inline content/en/docs/userguide/use_kmeshctl.md:124]: we can use ordered list under `kmeshctl waypoint`, it looks not very clear from https://deploy-preview-83--kmesh-net.netlify.app/en/docs/userguide/use_kmeshctl/

File: content/en/docs/userguide/use_kmeshctl.md (hunk 2 above)
Comment (YaoZengzeng) [reply content/en/docs/userguide/use_kmeshctl.md:58]: @LiZhenCheng9527 may need to optimize

**Comment by YaoZengzeng** [comment]
@hzxuzhonghu maybe we could merge first to facilitate users, and then further optimize it later.

**Review by hzxuzhonghu — APPROVED** [review]
/lgtm

**Pull request merged**